src/common/Cryptography/ARC4.cpp

```diff
@@ -17,34 +17,26 @@
 
 #include "ARC4.h"
 
-ARC4::ARC4(uint32 len) : m_ctx(EVP_CIPHER_CTX_new())
+Trinity::Crypto::ARC4::ARC4() : _ctx(EVP_CIPHER_CTX_new())
 {
-    EVP_CIPHER_CTX_init(m_ctx);
-    EVP_EncryptInit_ex(m_ctx, EVP_rc4(), nullptr, nullptr, nullptr);
-    EVP_CIPHER_CTX_set_key_length(m_ctx, len);
+    EVP_CIPHER_CTX_init(_ctx);
+    EVP_EncryptInit_ex(_ctx, EVP_rc4(), nullptr, nullptr, nullptr);
```

Comment:
Maybe some asserts sprinkled in here wouldn't be the worst idea.

Reply:
ah yes, i missed those